browser/js/content.js

```diff
@@ -60,61 +75,98 @@ function renderChat(parent, toUsername) {
   // TODO: Prevent navigation?
   const isLoggedIn = document.getElementsByClassName("logout").length > 0;
 
-  let nudgeHTML = '\
-    <p><label><input type="checkbox" name="keybase-nudge" checked /> <em>public</em> nudge (so they know about Keybase)</label></p>\
-    <p><textarea name="keybase-nudgetext">/u/'+ toUsername + ' - I left you an end-to-end encrypted reply in Keybase. https://keybase.io/reddit-crypto</textarea></p>\
-  ';
+  let nudgeHTML = `
+    <p>
+      <label><input type="checkbox" name="keybase-nudgecheck" checked /> Nudge publicly (reply in thread so they know about Keybase)</label>
+      <textarea name="keybase-nudgetext">/u/${toUsername} - I left you an end-to-end encrypted reply in Keybase. https://keybase.io/reddit-crypto</textarea>
+    </p>
+  `;
   if (!isLoggedIn) {
     // FIXME: Won't need this if we have a KeybaseBot PM'ing people?
-    nudgeHTML = '\
-      <p>You will need to let <a target="_blank" href="/u/'+ toUsername +'" class="reddit-user">/u/' + toUsername + '</a> know that they have a Keybase message waiting for them.</p>\
-      <p>Share this handy link: <a target="_blank" href="https://keybase.io/reddit-crypto">https://keybase.io/reddit-crypto</a></p>\
-    ';
+    nudgeHTML = `
```

Comment:
In the sense of templates getting inlined with other templates?

So far this is the only case, not sure if we need anything more formal yet.